### PR TITLE
Display entered word on typo

### PR DESCRIPTION
**Pull request by rwz**
When typo occurs, display entered word back at user. Feel free to reject, this is a super low-effort PR mainly made to scratch my own itch.

### CODE REVIEW

**Comment by rwz** [comment]
I wish there was a way to see what Vercel build is complaining about without registering for an account. I don't use TypeScript normally, so maybe my code is horribly broken, but it seems to work on the local dev server.

Not sure what's happening here.

**Comment by r3oath** [comment]
I'll take a look and see what's going wrong on the Vercel side.

File: components/word-visualiser.tsx [hunk 1]
```diff
@@ -58,7 +58,10 @@ const WordVisualiser = (): React.ReactElement | undefined => {
 		}
 
 		if (state.word.endTime !== undefined && !state.word.match) {
-			return 'Typo';
+			let output = 'Typo';
+			if (state.typoText != undefined && state.typoText != ' ')
+			output += `: ${state.typoText}`;
+			return output;
```
Comment (r3oath) [inline components/word-visualiser.tsx:64]: This doesn't follow the code style conventions; you'll need to add the missing braces on the if statement, add in the correct newline separators, and use strict equality checks:

```suggestion
			let output = 'Typo';

			if (state.typoText !== undefined && state.typoText !== ' ') {
				output += `: ${state.typoText}`;
			}

			return output;
```

Comment (r3oath) [reply components/word-visualiser.tsx:64]: `state.typoText` is also missing from the `useMemo` dependency array below.

**Review by r3oath — CHANGES_REQUESTED** [review]

**Comment by rwz** [comment]
Oh, I've discovered that I could run `npm run lint` and see what it doesn't like about my code. Also Vercel won't show details even to people with accounts, unless they belong to the right group, which seems unfortunate.

Anyways, current version seem to be passing.

**Comment by r3oath** [comment]
Yeah, that's expected; otherwise, I'd have to set up and pay for a Pro team account on Vercel. But everything should pass as long as you can run `npm run build` and `npm run lint` locally with no issues. If anything else is failing on the Vercel side, it's something that I would need to rectify personally.

Anyway, that's all looking good. I'll merge this in. Thanks for the contributions 👍

**Review by r3oath — APPROVED** [review]

**Pull request merged**